### PR TITLE
Added instructions for upgrade from 1.5 debian packages

### PR DESCRIPTION
**Pull request by scollazo**
Work in progress, I'm having problems running dashboard migrations.

### CODE REVIEW

File: admin-manual/installation/installation.rst [hunk 1]
```diff
@@ -76,30 +76,29 @@ Installing from packages
 
 Installing from packages is tested for both new installations and upgrading from version 1.4.
```
Comment (sromkey) [inline admin-manual/installation/installation.rst]: Line above should be changed to "1.5" (or 1.5.1?)

File: admin-manual/installation/installation.rst [hunk 2]
```diff
@@ -141,43 +142,51 @@ updating the database, say 'ok' to each of those steps. If you have set a
...
-   sudo apt-get install elasticsearch
-   sudo /etc/init.d/elasticsearch start
+   sudo apt-get -f install 
+   sudo dpkg-reconfigure archivematica-dashboard
+ 
+
+**Disable unused services**
 
-You will be prompted with questions about modifying configuration files.  If you have not made any modifications to your Elasticsearch configuration, it should be safe to use the new versions of the configuration files that come with Elasticsearch.
+Archivematica 1.6.0 uses nginx as http server, and gunicor as wsgi server. This means that some services used in Archivematica 1.5.0 should be stopped and disabled before perfoming the upgrade.
```
Comment (sromkey) [inline admin-manual/installation/installation.rst]: little typos: gunicorn, performing

File: admin-manual/installation/installation.rst [hunk 3]
```diff
@@ -141,43 +142,51 @@ updating the database, say 'ok' to each of those steps. If you have set a
...
-Instructions on how to upgrade can be found on the
-`Elasticsearch website <https://www.elastic.co/guide/en/elasticsearch/reference/1.3/setup-upgrade.html>`_.
-In general it should be possible to upgrade Elasticsearch on a standard Archivematica machine with the following commands:
+.. code:: bash
 
+   sudo apt-get remove python-pip
+   wget -O /tmp/get-pip.py https://bootstrap.pypa.io/get-pip.py
+   sudo python /tmp/get-pip.py
+ 
+ We can't do it in advance, as python-pip was a requirement for Archivematica 1.5. The upgrade can finish with:
```
Comment (sromkey) [inline admin-manual/installation/installation.rst]: Suggestion: "This step cannot be done in advance, as python-pip..."

**Comment by scollazo** [comment]
Addressed feedback from @sromkey

**Comment by scollazo** [comment]
this has been done already in other pr, closing

**Pull request closed without merging**